rest-service/manager_rest/dsl_functions.py

```diff
@@ -211,7 +211,7 @@ def get_group_capability(self, capability_path):
             return [caps for dep_id, caps in
                     sorted(capabilities.items(), key=lambda item: item[0])]
 
-    def get_sys(self, entity, prop):
+    def get_sys(self, entity, prop, subprop=None):
         deployment = self.sm.get(Deployment, self._deployment_id)
```

Comment:
We should probably complain if someone tries to pass subprop on something that doesn't support subprop.